src/components/panel/panel.stories.ts

```diff
@@ -170,50 +170,3 @@ export const darkThemeRTL_TestOnly = (): string =>
-
-export const noDoubleScrollbars_TestOnly = (): string => html`
-  <style>
-    #container {
-      display: flex;
-      max-height: 540px;
-      width: 300px;
-    }
-
-    .content {
-      height: 100%;
-      display: flex;
-      padding: 10px;
-      overflow-y: auto; /* Control scrollbar via child */
-    }
-  </style>
-  <div id="container">
-    <calcite-flow>
-      <calcite-panel heading="Example">
-        <div>### Stickied Content e.g. toolbar</div>
-        <div class="content">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sapien lectus, ultricies a molestie nec,
-          sollicitudin ac nulla. Pellentesque tincidunt malesuada arcu et placerat. In malesuada neque lectus, at congue
-          est malesuada quis. Proin tincidunt lacus laoreet mauris fringilla accumsan. Cras nec enim eu lectus suscipit
-          vestibulum a laoreet arcu. Duis posuere nunc vel enim blandit, nec vehicula orci aliquam. Vestibulum hendrerit
-          mi vel nisi posuere accumsan. Aenean efficitur est id cursus convallis. Morbi turpis ante, sodales eu tortor
-          eu, mattis bibendum purus. Morbi iaculis nisl nunc, quis accumsan quam laoreet vitae. Aliquam ex ligula,
-          ornare eu ex vitae, tincidunt venenatis lacus. Phasellus risus quam, elementum sed justo porttitor,
-          ullamcorper mattis nisl. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia
-          curae; Nulla non dui at metus porta lacinia congue sit amet quam. Mauris viverra diam neque, in blandit leo
-          vehicula et. Donec non purus vitae nunc tincidunt egestas. Nunc pretium enim magna, sed fringilla lacus
-          viverra in. Nam et pretium nisi. Ut bibendum, ipsum sit amet egestas hendrerit, quam orci sollicitudin purus,
-          sit amet finibus mauris erat in eros. Integer est dui, vehicula a ipsum id, pellentesque semper elit. Fusce
-          euismod volutpat eros vitae imperdiet. Nam suscipit lacus id posuere pharetra. Cras eros ipsum, feugiat non
-          leo non, ornare malesuada eros. Donec egestas purus non quam tempus commodo. Maecenas ex augue, euismod eget
-          magna in, dapibus fermentum felis. Phasellus justo felis, sollicitudin ut ex sed, lobortis scelerisque sem.
-          Pellentesque semper placerat velit, sit amet viverra tortor ultricies eu. Pellentesque habitant morbi
-          tristique senectus et netus et malesuada fames ac turpis egestas. Phasellus feugiat, augue in molestie
-          imperdiet, felis turpis facilisis tortor, at tempus purus risus et sapien. Fusce id nisi id orci elementum
-          sollicitudin. Nam id libero eu odio efficitur rutrum maximus porta lorem. Nunc tristique interdum augue,
-          sodales viverra lectus efficitur vitae. Nam molestie, neque consequat mollis pulvinar, sapien sem semper nunc,
-          et euismod enim sem vitae ligula.
-        </div>
-      </calcite-panel>
-    </calcite-flow>
-  </div>
-`;
```

Comment:
Was this only relevant with panel? Otherwise we should add it back but change it to flow-item

Reply:
yeah ill move it. 👍